### ISSUE TITLE
Add Cloudflare - Privacy Policy

### ISSUE THREAD

**Issue by martinratinaud** [OpenTermsArchive/contrib-declarations#56]
New service addition requested through the contribution tool

You can see the work done by the awesome contributor here:
https://opentermsarchive.org/en/contribute/service?documentType=Privacy%20Policy&expertMode=true&name=Cloudflare&selectedCss[]=.col-lg-8%20%3E%20.rich-text-renderer&step=2&url=https%3A%2F%2Fwww.cloudflare.com%2Fprivacypolicy%2F&expertMode=true

Or you can see the JSON generated here:
```json
{
  "name": "Cloudflare",
  "documents": {
    "Privacy Policy": {
      "fetch": "https://www.cloudflare.com/privacypolicy/",
      "select": [
        ".col-lg-8 > .rich-text-renderer"
      ]
    }
  }
}
```

You will need to create the following file in the root of the project: `services/Cloudflare.json`

Transfered from https://github.com/ambanum/OpenTermsArchive/issues/627

**Comment by Kissaki** [comment]
This was implemented in #766 and should be closed

**Comment by MattiSG** [comment]
Thanks @Kissaki for the triaging work 🙏
